emhttp/plugins/dynamix/styles/themes/README.md

```diff
@@ -0,0 +1,258 @@
+# Dynamix Theme System Documentation
+
+## Overview
+
+The Dynamix theme system uses CSS variables (custom properties) to create a flexible and maintainable theming solution. The system is built on several layers that work together to provide consistent styling across the webGUI.
+
+## Core Components
+
+### 1. Base Color Palette (`default-color-palette.css`)
+
+- Defines fundamental color variables used throughout the system
+- Contains standardized color scales (e.g., `--gray-100` through `--gray-900`)
+- Includes opacity variants for common colors (e.g., `--white-opacity-10`)
+
+### 2. Base Styles (`default-base.css`)
+
+- Provides foundational styling for the entire application
+- Defines core layout and component styles
+- Implements responsive design patterns
+- Key sections include:
+  - Typography and text styles
+  - Form elements (inputs, buttons, selects)
+  - Layout components (header, menu, footer)
+  - Grid and table structures
+  - Utility classes and animations
+  - Theme-specific modifiers (e.g., `.Theme--sidebar`)
+
+#### Base Style Categories
+
+1. **Core Elements**
+   - HTML/Body defaults
+   - Typography scales
+   - Link styles
+   - Form elements
+
+2. **Layout Components**
+   - Header structure
+   - Navigation system
+   - Content areas
+   - Footer positioning
+
+3. **Interactive Elements**
+   - Button states
+   - Form input behaviors
+   - Hover effects
+   - Focus states
+
+4. **Utility Classes**
+   - Color modifiers (`.green`, `.red`, etc.)
+   - Status indicators
+   - Usage bars and meters
+   - Helper classes
+
+5. **Responsive Patterns**
+   - Media queries for different screen sizes
+   - Flexible layouts
+   - Mobile adaptations
+
+### 3. Theme Files (`themes/*.css`)
+
+- Located in `emhttp/plugins/dynamix/styles/themes/`
+- Each theme (azure, black, gray, white) defines its specific color scheme
+- Uses variables from the base color palette
+- Sets theme-specific variables used by component sheets
+
+### 4. Component Sheets (`sheets/*.css`)
+
+Component sheets are located in `emhttp/plugins/dynamix/styles/sheets/` and follow these patterns:
+
+#### Variable Declaration
+
+```css
+:root {
+    --component-element-property-color: var(--color-palette-variable);
+}
+```
+
+#### Theme-Specific Overrides
+
+```css
+.Theme--{themename}:root {
+    --component-element-property-color: var(--theme-specific-variable);
+}
+```
+
+## Component Sheet Types
+
+### 1. Feature-Specific Sheets
+
+Files like `Browse.css`, `DashStats.css` that define styles for specific features:
+
+- Declare component-specific variables
+- Use theme variables for consistent styling
+- Include theme-specific overrides when needed
+
+### 2. Theme-Variant Sheets
+
+Files like `Eth0-azure.css`, `Wireless-black.css` that provide theme-specific styles:
+
+- Contain minimal overrides for specific components in specific themes
+- Usually consist of a few critical variable changes
```

Comment:
This needs to be removed. AI wrote this before I got rid of `Eth0` & `Wireless`.